introduction.md

```diff
@@ -5,6 +5,11 @@ Hello, sexy sensei, Simon-san desu ka!
  ( a a )-.___...-'/
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?